src/main/java/org/commonjava/event/storage/BatchCleanupRequest.java

```diff
@@ -0,0 +1,39 @@
+package org.commonjava.event.storage;
+
+import java.util.Set;
+
+/**
+ * Cleanup multiple paths from multiple filesystems.
+ */
+public class BatchCleanupRequest
+{
+    private Set<String> paths;
+
+    private Set<String> filesystems;
```

Comment:
So is this many-to-many ?

Reply:
Yes.

Reply:
I use two for loops, like for{ for{} }, to delete files.

Reply:
I see. thanks.